ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -98,6 +98,7 @@
 skipif_aws_creds_are_missing = pytest.mark.skipif(
     (
         load_auth_config().get('AUTH', {}).get('AWS', {}).get('AWS_ACCESS_KEY_ID') is None
+        and 'AWS_ACCESS_KEY_ID' not in os.environ
```

Comment:
If you dont mind waiting for a day, we can fix it properly? but I will approve if its blocking

Reply:
@MeridianExplorer if we can use exception and skip that will be nice without additional environment variables.

Reply:
What's not proper about this?

Reply:
I was thinking if we can avoid environment variable since its valid for many other aws environment, we just hit this today in other dev environment and I filed an issue too.

Reply:
This issue is handled in PR https://github.com/red-hat-storage/ocs-ci/pull/2700